denlillemand/week4/ass5/app/mainlist.jsx~

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
```

Comment:
Looks like a left over file from an editor. Should be removed from source control